packages/cli/src/config/webpack.config.common.js

```diff
@@ -138,6 +138,8 @@ module.exports = ({ config, context }) => {
       }]
     },
 
+    optimization: config.optimization,
```

Comment:
let's back out exposing this to the user for now
https://github.com/ProjectEvergreen/greenwood/pull/265/files#r355764479